DTALauncherStub.csproj

```diff
@@ -1,145 +1,39 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{7477EAC1-2302-4311-9F7D-2D65C016C857}</ProjectGuid>
+    <TargetFramework>net7.0-windows</TargetFramework>
     <OutputType>WinExe</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <UseWindowsForms>true</UseWindowsForms>
+    <Title>DTA Launcher</Title>
+    <Description>Launcher for Dawn of the Tiberium Age CnCNet Client</Description>
+    <Company>CnCNet</Company>
+    <Product>Dawn of the Tiberium Age</Product>
+    <Copyright>Copyright © CnCNet, Rampastring 2011-2022</Copyright>
+    <Trademark></Trademark>
+    <Version>2.0.0.1</Version>
+    <FileVersion>2.0.0.1</FileVersion>
+    <InformationalVersion>2.0.0.1</InformationalVersion>
```

Comment:
Thought we generate this through GitVersion?

Reply:
I initially copied over the values from the original `AssemblyInfo.cs`. I removed them as they are indeed generated during build.